std/format/read.d

```diff
@@ -1,35 +1,69 @@
 // Written in the D programming language.
 
 /**
-   This is a submodule of $(MREF std, format).
-   It provides some helpful tools.
-
-   Copyright: Copyright The D Language Foundation 2000-2013.
-
-   License: $(HTTP boost.org/LICENSE_1_0.txt, Boost License 1.0).
-
-   Authors: $(HTTP walterbright.com, Walter Bright), $(HTTP erdani.com,
-   Andrei Alexandrescu), and Kenji Hara
-
-   Source: $(PHOBOSSRC std/format/read.d)
+This is a submodule of $(MREF std, format).
+
+It provides two functions for reading formatted $(LREF unformatValue)
+reads a single value, while $(LREF formattedRead) reads several
+values at once and matches the characters found between format
+specifiers.
+
+For reading purpose, only part of the possibilities of $(MREF_ALTTEXT
+format strings, std,format) is used: The parameters are almost
+completely ignored. The only exception is a single $(B '*'): It
+denotes that the read value is not assigned to a variable, but is
+just discarded. A $(B '*') can only be applied to a $(B 'c'),
+$(B 'd') or $(B 'u') format character.
```

Comment:
I don't understand this paragraph.

Reply:
Maybe I should add an example, probably something like this (taken from the non public unittests):

```
    import std.typecons : Tuple;

    Tuple!(int, float) t;
    char[] line = "1 7643 2.125".dup;
    formattedRead(line, "%s %*u %s", t);
    assert(t[0] == 1 && t[1] == 2.125);
```

Reply:
Maybe this shouldn't the emphasized here, but rather explained at unittest site.

Reply:
As this only applies to `formattedWrite` anyway, I put the example there and added an explaining line in the docs there. I kept part of this paragraph, because I think it should be known, that only part of the grammar applies for reading.